Update Razor to 6.0.0-alpha1.20521.3 (#4135)
OmniSharp/omnisharp-vscode: CHANGELOG.md

## 1.23.5 (Not yet released)
* Set meaning of UseGlobalMono "auto" to "never" since Mono 6.12.0 still ships with MSBuild 16.7 (PR: [#4130](https://github.com/OmniSharp/omnisharp-vscode/pull/4130))
* Ensure that the rename identifier and run code action providers do not apply changes twice PR: [4133](https://github.com/OmniSharp/omnisharp-vscode/pull/4133)
* Update Razor to 6.0.0-alpha1.20521.3:

>   * HTML colorization is now computed meaning it should be "more correct" for non-C# portions of the document.

Reviewer: ```suggestion
  * Improvements to HTML colorization for non-C# portions of the document.
```
